File: backend-auth/src/main/java/cc/jktu/apps/auth/dao/mapper/UserMapper.java
package cc.jktu.apps.auth.dao.mapper;

import cc.jktu.apps.common.dao.entity.UserEntity;
import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface UserMapper extends BaseMapper<UserEntity> {
}

File: backend-auth/src/main/kotlin/cc/jktu/apps/auth/dao/mapper/UserMapper.kt
package cc.jktu.apps.auth.dao.mapper

import cc.jktu.apps.common.dao.entity.UserEntity
import com.baomidou.mybatisplus.core.mapper.BaseMapper
import org.apache.ibatis.annotations.Mapper

@Mapper
interface UserMapper : BaseMapper<UserEntity>
